tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -79,6 +79,24 @@ test.serial('error when provider info is missing', async (t) => {
   }
 });
 
+test.serial('error when collection info is not provided in the event', async (t) => {
```

Comment:
It looks like the title for this test is wrong since, if I understand this change right, this test should _not_ cause an error.